src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -221,6 +227,10 @@ public async Task Read()
             if (pageSize == 0)
                 return rangeResultDictionary;
 
+            Dictionary<string, BlittableJsonReaderObject> includes = null;
+            if (includeDocument || includeTags)
```

Comment:
can you wrap this inside a command like we have with other includes?

Reply:
done